server/grpc_service.go

```diff
@@ -1204,7 +1204,7 @@ func (s *GrpcServer) ScatterRegion(ctx context.Context, request *pdpb.ScatterReg
 		}, nil
 	}
 
-	region := rc.GetRegion(request.GetRegion().GetId())
+	region := rc.GetRegion(request.GetRegionId())
```

Comment:
I think we should support both fields and completely remove the deprecated one in a future release.

Reply:
the new field is `GetRegionsId`, then `regionID` deprecated in https://github.com/pingcap/kvproto/pull/686. Both field `regionID` and `regionsID` are supported. here is incorrectly use `Region` field.